frameworks/AutoGluonTS/__init__.py

```diff
@@ -19,9 +19,11 @@ def run(dataset: Dataset, config: TaskConfig):
             name=dataset.target.name,
             classes=dataset.target.values
         ),
-        problem_type=dataset.type.name  # AutoGluon problem_type is using same names as amlb.data.DatasetType
+        problem_type=dataset.type.name,  # AutoGluon problem_type is using same names as amlb.data.DatasetType
+        timestamp_column=dataset.timestamp_column if dataset.timestamp_column is not None else None,
+        id_column=dataset.id_column if dataset.id_column is not None else None,
+        prediction_length=dataset.prediction_length if dataset.prediction_length is not None else None
```

Comment:
Should we instead check if `dataset.timestamp_column` exists rather than if it is not None? ditto for the others.

Reply:
Currently these if/else don't actually do anything